.circleci/config.yml

```diff
@@ -95,9 +89,13 @@ executors:
   puppeteer-executor:
     docker:
       - image: << pipeline.parameters.workbench-image >>
+    resource_class: medium+
```

Comment:
Did this have a noticeable impact? I'm still very skeptical that more resources are necessary; unless you're actually seeing OOM errors, or steps like starting the UI taking too long.

Reply:
Few minutes faster. I will change to use default (resource_class: medium) because it's more $ using medium+.

<img width="853" alt="Screen Shot 2021-02-16 at 5 13 41 PM" src="https://user-images.githubusercontent.com/35533885/108128173-589df500-707a-11eb-9e41-49cc8f8773af.png">